src/SignhostAPIClient/Rest/SignHostApiClient.cs

```diff
@@ -26,6 +31,65 @@ public class SignHostApiClient
 		private readonly ISignHostApiClientSettings settings;
 		private readonly HttpClient client;
 
+		public bool IsPostbackChecksumValid(
+			IDictionary<string, string[]> headers,
+			string body,
+			out Transaction postbackTransaction)
+		{
+			postbackTransaction = null;
+			string[] postbackChecksumArray;
+			string postbackChecksum;
+			string calculatedChecksum;
+			PostbackTransaction postback;
+
+			var jsonSettings = new JsonSerializerSettings() {
+				Error = delegate(object sender,
+					Newtonsoft.Json.Serialization.ErrorEventArgs args) {
+					args.ErrorContext.Handled = true;
+				}
+			};
+
+			postback = JsonConvert.DeserializeObject<PostbackTransaction>(body);
+			postbackTransaction = postback;
+
+			if (headers.TryGetValue("Checksum", out postbackChecksumArray)) {
+				postbackChecksum = postbackChecksumArray.First();
+			}
+			else {
+				postbackChecksum = postback.Checksum;
+			}
+
+			if (!string.IsNullOrEmpty(postbackTransaction.Id) &&
+				!string.IsNullOrEmpty(postbackChecksum)) 
+			{
```

Comment:
In case of multiline if statements format it like this:
```
if (!string.IsNullOrEmpty(postbackTransaction.Id) &&
    !string.IsNullOrEmpty(postbackChecksum)
) {
```

Reply:
Can't fix this. Seems like a MacOS VS bug.